apps/cli/src/compose/rollups/docker-compose-espresso.yaml

```diff
@@ -14,36 +15,28 @@ services:
                 condition: service_healthy
         environment:
             ESPRESSO_SEQUENCER_L1_PROVIDER: ${CARTESI_BLOCKCHAIN_HTTP_ENDPOINT:-http://anvil:8545}
-            ESPRESSO_SEQUENCER_API_PORT: 8770
-            ESPRESSO_BUILDER_PORT: 8771
-            ESPRESSO_PROVER_PORT: 8772
+            ESPRESSO_SEQUENCER_API_PORT: 24000
+            ESPRESSO_BUILDER_PORT: 23000
             ESPRESSO_DEV_NODE_PORT: 20000
-            ESPRESSO_SEQUENCER_POSTGRES_HOST: database
-            ESPRESSO_SEQUENCER_POSTGRES_PORT: 5432
-            ESPRESSO_SEQUENCER_POSTGRES_USER: postgres
-            ESPRESSO_SEQUENCER_POSTGRES_PASSWORD: password
-            ESPRESSO_SEQUENCER_POSTGRES_DATABASE: espresso
             ESPRESSO_SEQUENCER_ETH_MNEMONIC: ${CARTESI_AUTH_MNEMONIC:-test test test test test test test test test test test junk}
-            ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: "1s"
-            ESPRESSO_STATE_PROVER_UPDATE_INTERVAL: "1s"
+            ESPRESSO_SEQUENCER_L1_POLLING_INTERVAL: "200ms"
```

Comment:
Isn't his too extreme?

Reply:
I'm using default values as a reference. 

Not sure what's the best values for our environment.

Reply:
Polling 5 times per second seems too much.
@fabiooshiro @lynoferraz is this necessary?

Believe it or not this contributes to drain the developer's laptop battery.

Reply:
If I understand correctly, this is controlling how often the local Espresso Dev Node will be polling the internal Avail instance right? So it's all local stuff. Still, what's the default block time for our internal Avail, 2s? If that's it, a 1s polling interval does seem enough.

Reply:
CLI default block time is 5s